style.css

```diff
@@ -101,3 +100,235 @@ body {
   display: flex;
   gap: 20px;
 }
+
+/* social media icons----------------------------------------------------------------------- */
+.Workssection {
+  border-radius: 8px;
+}
```

Comment:
- [ ] For proper organizing of the content in this tag, could you consider styling it with grid as display property? You can read about the usefulness [here](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)